SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Utils/RPCContainer/RPCContainerTest.cpp

```diff
@@ -215,8 +215,15 @@ RPCCONTAINER_TEST(GIVEN_a_container_with_one_value_WHEN_processing_after_RPCQueu
 
 	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
 	check(SpatialGDKSettings != nullptr);
-	FPlatformProcess::Sleep(SpatialGDKSettings->RPCQueueWarningDefaultTimeout);
+	FPlatformProcess::Sleep(SpatialGDKSettings->RPCQueueWarningDefaultTimeoutSeconds);
 	RPCs.ProcessRPCs();
+	constexpr float ErrorRetryWaitTime = 0.5f;
+	int NrRetries = 0;
+	while (!HasMetExpectedErrors() && NrRetries <= 10)
+	{
+		NrRetries++;
+		FPlatformProcess::Sleep(ErrorRetryWaitTime);
```

Comment:
Its pretty unusual for a unit test to work with timeouts, is it worth moving this SpatialFunctionalTest where we have better machinery for this? Also any idea why we need this retry?

Reply:
We've hypothesized that the reason for this flake is the error is logged on a different thread than the test, and it doesn't have time to log the error before the test finishes. Previously I had added just a sleep but this is a bit cleaner.

Reply:
(The flake is Error: Expected Error or Warning matching 'Unresolved Parameters' to occur 1 times with Contains match type, but it was found 0 time(s))